docs/Architecture/CBS.md

```diff
@@ -0,0 +1,69 @@
+## Current Processes
+
+### CBS: pulls data from email to s3
```

Comment:
I did not find this step in ANYWAY Media Flow (Miro)

Reply:
Thanks! I modified it